### PR TITLE
FIX DOCUMENT_REGEX

### PR DESCRIPTION
**Pull request by jayk09**
Fixes i18n init for a service worker:
https://github.com/kaisermann/svelte-i18n/issues/100

### CODE REVIEW

File: src/i18n.js [hunk 1]
```diff
@@ -41,7 +41,8 @@ export function startClient() {
 	});
 }
 
-const DOCUMENT_REGEX = /^([^.?#@]+)?([?#](.+)?)?$/;
+// init only for routes (urls with no extensions / .js, .css, ... /) and a service worker
```
Comment (kaisermann) [inline src/i18n.js:44]:
```suggestion
// init only for routes (urls with no extensions such as .js, .css, etc) and for service worker
```

File: src/i18n.js [hunk 2]
```diff
@@ -41,7 +41,8 @@ export function startClient() {
...
+const DOCUMENT_REGEX = /(\/([\w-]+)?|^\/service-worker-index\.html)($|[?#]([\w-]+)?)/;
```
Comment (kaisermann) [inline src/i18n.js:45]: I'm not sure about this, but is it possible for the service-worker URL to have something after the `.html`?

```suggestion
const DOCUMENT_REGEX = /(^([^.?#@]+)?([?#](.+)?)?|service-worker.*?\.html)$/;
```

**Review by kaisermann — CHANGES_REQUESTED** [review]

File: src/i18n.js (hunk 2 above)
Comment (jayk09) [reply src/i18n.js:45]: I'm not sure either... Anyway let's keep your solution...)

**Review by kaisermann — APPROVED** [review]

**Comment by kaisermann** [comment]
Thanks 😁 !

**Pull request merged**